examples/nuxtapp/pages/kbWhisper.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { kbWhisperlocal } from '../../../lib/src/kbWhisperLocal'
```

Comment:
Should import as package and not with relative path to make example more similar to how a third party would consume or use the lib module.
This suggestion implies kbWhisperLocal is also moved to the recognitionService folder.
```suggestion
import { kbWhisperlocal } from 'speech-utils/recognitionService/kbWhisperLocal'
```